pipeline/train/train.sh

```diff
@@ -93,6 +93,12 @@ paste <(${COMPRESSION_CMD} -dc "${valid_set_prefix}.${src}.${ARTIFACT_EXT}") \
       >"${valid_tsv_dataset}"
 
 
+# we run a CPU version of Marian in tests and it does not work with these arguments
+if [[ -z ${USE_CPU+x} ]]; then
```

Comment:
What's the purpose of using the `x` default value if this is set? It seems like you could just use plain `[[ -z ${USE_CPU} ]]` here?

Reply:
it fails without the default value if the variable is not set